Sources/BraintreePayPal/BTPayPalVaultRequest.swift

```diff
@@ -11,6 +11,9 @@ import BraintreeCore
 
     /// Optional: Offers PayPal Credit if the customer qualifies. Defaults to `false`.
     public var offerCredit: Bool
+    
+    /// Optional: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
+    public var payerEmail: String?
```

Comment:
We call this `userAuthenticationEmail` for the Native offering ([here](https://github.com/braintree/braintree_ios/blob/075220827e2df164c8230a8ed43b81dba7a402d3/Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutRequest.swift#L16-L17)).

Do folks have a naming preference here?

Reply:
I feel like `userAuthenticationEmail` could make more sense vs `payerEmail`. The latter doesn't indicate as clearly that the email is used to attempt to authenticate quicker, imo.

Though looking through other modules, we don't tend to prefix payer/user/customer, for example `BTPayPalAccountNonce` [here](https://github.com/braintree/braintree_ios/blob/075220827e2df164c8230a8ed43b81dba7a402d3/Sources/BraintreePayPal/BTPayPalAccountNonce.swift#L10-L20) and `BTLocalPaymentRequest` [here](https://github.com/braintree/braintree_ios/blob/075220827e2df164c8230a8ed43b81dba7a402d3/Sources/BraintreeLocalPayment/BTLocalPaymentRequest.swift#L41-L51). So we _could_ go with just `authenticationEmail` potentially. What do you think?

Reply:
I think matching the naming from the Native PayPal module makes sense.

I feel having `payer` or `buyer` adds some clarity in the param name, but also good to align with the other properties we have that have it omitted.

Reply:
Updated to match the NXO module!